docs/source/pythonapi/base.rst

```diff
@@ -208,3 +197,53 @@ Various classes may be created when performing tally slicing and/or arithmetic:
    openmc.arithmetic.AggregateScore
    openmc.arithmetic.AggregateNuclide
    openmc.arithmetic.AggregateFilter
+
+Coarse Mesh Finite Difference Acceleration
+------------------------------------------
+
+.. autosummary::
+   :toctree: generated
+   :nosignatures:
+   :template: myclass.rst
+
+   openmc.CMFDMesh
+   openmc.CMFDRun
+
+CMFD is implemented in OpenMC and allows users to accelerate fission source
+convergence during inactive neutron batches. To run CMFD, the CMFDRun class should
+be used. The following properties can be set through the CMFDRun class:
+
+.. autosummary::
+   :toctree: generated
+   :nosignatures:
+   :template: myfunction.rst
+
+   openmc.CMFDRun.cmfd_begin
+   openmc.CMFDRun.dhat_reset
+   openmc.CMFDRun.cmfd_display
+   openmc.CMFDRun.cmfd_downscatter
+   openmc.CMFDRun.cmfd_feedback
+   openmc.CMFDRun.cmfd_ktol
+   openmc.CMFDRun.cmfd_mesh
+   openmc.CMFDRun.norm
+   openmc.CMFDRun.cmfd_adjoint_type
+   openmc.CMFDRun.cmfd_power_monitor
+   openmc.CMFDRun.cmfd_run_adjoint
+   openmc.CMFDRun.cmfd_shift
+   openmc.CMFDRun.cmfd_stol
+   openmc.CMFDRun.cmfd_spectral
+   openmc.CMFDRun.cmfd_reset
+   openmc.CMFDRun.cmfd_write_matrices
+   openmc.CMFDRun.gauss_seidel_tolerance
```

Comment:
All of these are properties, so they won't show up correctly when listed as functions. In fact, there's no reason to list them explicitly: the autogenerated documentation for the `CMFDRun` class will show all the properties.